packages/calcite-components/src/components/tile/tile.e2e.ts

```diff
@@ -1,32 +1,124 @@
 import { newE2EPage } from "@stencil/core/testing";
-import { accessible, defaults, disabled, reflects, renders, slots, hidden } from "../../tests/commonTests";
+import { accessible, defaults, disabled, focusable, hidden, reflects, renders, slots } from "../../tests/commonTests";
+import { html } from "../../../support/formatting";
 import { SLOTS } from "./resources";
 
 describe("calcite-tile", () => {
-  describe("renders", () => {
-    renders("calcite-tile", { display: "inline-block" });
+  describe("accessibility", () => {
+    accessible(html` <calcite-tile></calcite-tile> `, "accessible without label");
+    accessible(html` <calcite-tile label="my-tile"></calcite-tile> `, "accessible with label only");
+    accessible(
+      html` <calcite-tile label="my-tile" selection-mode="single"></calcite-tile> `,
+      "accessible in single selection-mode",
+    );
+    accessible(
+      html` <calcite-tile label="my-tile" selection-mode="single-persist"></calcite-tile> `,
+      "accessible in single-persist selection-mode",
+    );
+    accessible(
+      html` <calcite-tile label="my-tile" selection-mode="multiple"></calcite-tile> `,
+      "accessible in multiple selection-mode",
+    );
+    accessible(html` <calcite-tile href="#" heading="My link"></calcite-tile> `, "accessible as link with heading");
+    accessible(
+      html` <calcite-tile href="#" description="My link"></calcite-tile> `,
+      "accessible as link with description",
+    );
+    accessible(
+      html` <calcite-tile label="my-tile" href="#" heading="My link"></calcite-tile> `,
+      "accessible as link with heading and label",
+    );
+    accessible(
+      html` <calcite-tile label="my-tile" href="#" description="My link"></calcite-tile> `,
+      "accessible as link with description and label",
+    );
   });
 
-  describe("honors hidden attribute", () => {
-    hidden("calcite-tile");
-  });
+  describe("click", () => {
+    it("should receive focus when clicked", async () => {
```

Comment:
Shouldn't this fail when the tile is not interactive?

Reply:
I need to understand this pattern better.  This same behavior happens in Chip as well, and because of the confusing interplay between the `interactive` and `disabled` properties, I'm not sure what the correct state of these should be with a Tile in its most basic form as a standalone component.  I'll reach out to you more directly with questions on this.

Reply:
I figured out that `tabIndex` should be set to `undefined` instead of `-1` when the `interactive` prop is false.  This will ensure that clicks don't focus the element.  I updated this test to have the reverse expectation and also added an additional one to assert it works when it is interactive.